helm/prometheus-rules/templates/alerting-rules/capi.rules.yml

```diff
@@ -0,0 +1,60 @@
+{{- if or (eq .Values.managementCluster.provider.kind "openstack") }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels: {{- include "labels.common" . | nindent 4}}
+  name: capi.rules
+  namespace: {{.Values.namespace}}
+spec:
+  groups:
+    - name: coredns
+      rules:
+        - alert: MachineUnhealthyPhase
+          annotations:
+            description: "{{`Machine {{ $labels.namespace}}/{{ $labels.name }} stuck in phase {{ $labels.phase }} for more than 15 minutes.`}}"
+          expr: capi_machine_status_phase{phase!~"Running|Provisioning"} > 0
```

Comment:
I think stuck in `Provisioning` is also not good, no? But if it's excluded then possibly duration has to be increased.

Reply:
Let's drop `Provisioning` for now and once we have some data we can adjust the expression.